libs/pubsub/query/oldquery/query_test.go

```diff
@@ -0,0 +1,205 @@
+			map[string][]string{"oranges.kg": {"3"}, "watermellons.kg": {"12"}},
+			true,
+		},
+		{"peaches.kg < 4", map[string][]string{"peaches.kg": {"5"}}, false},
+		{
+			"tx.date > DATE 2017-01-01",
+			map[string][]string{"tx.date": {time.Now().Format(query.DateLayout)}},
+			true,
+		},
+		{"tx.date = DATE 2017-01-01", map[string][]string{"tx.date": {txDate}}, true},
+		{"tx.date = DATE 2018-01-01", map[string][]string{"tx.date": {txDate}}, false},
+		{
+			"tx.time >= TIME 2013-05-03T14:45:00Z",
+			map[string][]string{"tx.time": {time.Now().Format(query.TimeLayout)}},
+			true,
+		},
+		{"tx.time = TIME 2013-05-03T14:45:00Z", map[string][]string{"tx.time": {txTime}}, false},
+		{"abci.owner.name CONTAINS 'Igor'", map[string][]string{"abci.owner.name": {"Igor,Ivan"}}, true},
+		{"abci.owner.name CONTAINS 'Igor'", map[string][]string{"abci.owner.name": {"Pavel,Ivan"}}, false},
+		{"abci.owner.name = 'Igor'", map[string][]string{"abci.owner.name": {"Igor", "Ivan"}}, true},
+		{
+			"abci.owner.name = 'Ivan'",
+			map[string][]string{"abci.owner.name": {"Igor", "Ivan"}},
+			true,
+		},
+		{
+			"abci.owner.name = 'Ivan' AND abci.owner.name = 'Igor'",
+			map[string][]string{"abci.owner.name": {"Igor", "Ivan"}},
+			true,
+		},
+		{
+			"abci.owner.name = 'Ivan' AND abci.owner.name = 'John'",
+			map[string][]string{"abci.owner.name": {"Igor", "Ivan"}},
+			false,
+		},
+		{
+			"tm.events.type='NewBlock'",
+			map[string][]string{"tm.events.type": {"NewBlock"}, "app.name": {"fuzzed"}},
+			true,
+		},
+		{
+			"app.name = 'fuzzed'",
+			map[string][]string{"tm.events.type": {"NewBlock"}, "app.name": {"fuzzed"}},
+			true,
+		},
+		{
+			"tm.events.type='NewBlock' AND app.name = 'fuzzed'",
+			map[string][]string{"tm.events.type": {"NewBlock"}, "app.name": {"fuzzed"}},
+			true,
+		},
+		{
+			"tm.events.type='NewHeader' AND app.name = 'fuzzed'",
+			map[string][]string{"tm.events.type": {"NewBlock"}, "app.name": {"fuzzed"}},
+			false,
+		},
+		{"slash EXISTS",
+			map[string][]string{"slash.reason": {"missing_signature"}, "slash.power": {"6000"}},
+			true,
+		},
+		{"sl EXISTS",
+			map[string][]string{"slash.reason": {"missing_signature"}, "slash.power": {"6000"}},
+			true,
+		},
+		{"slash EXISTS",
+			map[string][]string{"transfer.recipient": {"cosmos1gu6y2a0ffteesyeyeesk23082c6998xyzmt9mz"},
+				"transfer.sender": {"cosmos1crje20aj4gxdtyct7z3knxqry2jqt2fuaey6u5"}},
+			false,
+		},
+		{"slash.reason EXISTS AND slash.power > 1000",
+			map[string][]string{"slash.reason": {"missing_signature"}, "slash.power": {"6000"}},
+			true,
+		},
+		{"slash.reason EXISTS AND slash.power > 1000",
+			map[string][]string{"slash.reason": {"missing_signature"}, "slash.power": {"500"}},
+			false,
+		},
+		{"slash.reason EXISTS",
+			map[string][]string{"transfer.recipient": {"cosmos1gu6y2a0ffteesyeyeesk23082c6998xyzmt9mz"},
+				"transfer.sender": {"cosmos1crje20aj4gxdtyct7z3knxqry2jqt2fuaey6u5"}},
+			false,
+		},
+	}
+
+	for _, tc := range testCases {
```

Comment:
subtests?

Reply:
These are the old implementation tests. I'm just going to delete them.